checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
+
+class Vigenere(Test):
```

Comment:
Probably best not to use `Suite` because the `TestSuite` class in `unittest2` already exists and serves a different function. Could rename `Test` to `TestCase` though if that's preferable.